config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index], controller: 'users/posts'
```

Comment:
- Good job done so far, please add a `show` route for the post controller, this should be handled by the `show` action in the `post` controller. Please fix this 🙏 
- Also, As I mentioned, the `users/posts` controller is not needed, so please remove the reference to that file 👍 👍